lib/fatal_error/fatal_error.c

```diff
@@ -21,6 +22,10 @@ void k_sys_fatal_error_handler(unsigned int reason,
 
 	LOG_PANIC();
 
+	if (IS_ENABLED(CONFIG_IS_SPM) && IS_ENABLED(CONFIG_SPM_NS_DEBUG_LOGS_DUMP)) {
```

Comment:
I don't want to start a war here, but our .clang-format is still 80 chars and AFAIU it is preferred to stay within 80 unless it makes the code less readable. IMO we should break this line to fit within 80, but feel free to ignore.